airbyte-integrations/connectors/source-instagram/source_instagram/streams.py

```diff
@@ -124,10 +133,12 @@ def read_records(
 class UserLifetimeInsights(InstagramStream):
     """Docs: https://developers.facebook.com/docs/instagram-api/reference/ig-user/insights"""
 
-    primary_key = None
+    primary_key = ["business_account_id", "metric", "date"]
```

Comment:
Since I don't know the domain this might be a dumb question : should the page_id be in the primary key as well? Can we have the same metric on the same date for different pages for a specific account id? 

The same question applies to all insight streams

Reply:
The relationship between a Facebook Page (`page_id`) and an Instagram Business Account (`business_account_id`) is one-to-one, as shown [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-instagram/source_instagram/api.py#L84-L89).

However, I realized that I still need to update the primary key to include a date for MediaInsights. I will address this in the next commit.